Automattic/wp-calypso
# Woo: Product action-list support for variation deletes

This adds support to the edit state and action-list for
variation deletes. It omits variations that have a delete edit from
the combined view of edits and API data, and it adds the
deletes to the action list.

*Note for testing: Edit state is not being cleared completely yet, so you'll need to do a browser refresh after you save any changes.*

To test:

1. `npm test`
2. `npm start`
3. Visit `http://calypso.localhost:3000/store/product/<site_id>/<product_id>`
4. Ensure product is variable with several variations (edit and save them if needed, then refresh).
5. Click the "x" next to a variation type option.
6. Observe the variation type option disappears, and the variations involved with that option disappear as well.
7. Click "Update" to save changes via API.
8. Verify that the variation type option no longer exists either by refreshing the page or checking wp-admin interface.
